org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/ArrayTest.java

```diff
@@ -407,12 +397,8 @@ public void test013() {
 // Check return type of array#clone()
 public void test014() throws Exception {
 	Map optionsMap = getCompilerOptions();
-	CompilerOptions options = new CompilerOptions(optionsMap);
-	if (options.complianceLevel > ClassFileConstants.JDK1_4) {
-		// check that #clone() return type is changed ONLY from -source 1.5 only (independant from compliance level)
-		optionsMap.put(CompilerOptions.OPTION_Source, CompilerOptions.VERSION_1_4);
-	}
-	this.runNegativeTest(
+	// check that #clone() return type is changed ONLY from -source 1.5 only (independant from compliance level)
+	this.runConformTest(
```

Comment:
Please remove the comment // check that ... and also the bugzilla reference just above the test - they are referring to 1.4 only behavior - leaving those references can be confusing.

Reply:
Removed